scripts/setup.cls

```diff
@@ -1,17 +1,46 @@
 public static void qaSetup() {
-  enableCourseConnections();
+    configQACustomSettings();
```

Comment:
@robertbwatson not sure if you are doing the same work to enable settings for dev/qa configs. but fyi..

Reply:
@xinnanlisf  No worries - I modified the execute_install_apex task (or execute_dev_apex) in #945 so should be fine here.